packages/vscode-extension/src/controls/sampleGallery/SampleGallery.tsx

```diff
@@ -194,6 +201,9 @@ export default class SampleGallery extends React.Component<unknown, SampleGaller
           });
         }
         break;
+      case Commands.OpenDesignatedSample:
+        this.onSampleSelected(event.data.sampleId, TelemetryTriggerFrom.ExternalUrl);
```

Comment:
If the `sampleId` in external link is incorrect, there won't be any warnings. Users will see the sample gallery page instead of detail page.

Reply:
Yeah, I would see this as a fallback behavior. We will validate our links before putting them in docs. If anything goes wrong, directing to gallery page seems like the right behavior.